plugins/dashboard/manafeed.go

```diff
@@ -149,7 +149,7 @@ func sendManaMapOnline() {
 		Type: MsgTypeManaMapOnline,
 		Data: accessPayload,
 	})
-	consensusManaList, err := manaPlugin.GetOnlineNodes(mana.AccessMana, mana.Mixed)
+	consensusManaList, err := manaPlugin.GetOnlineNodes(mana.AccessMana)
```

Comment:
Should this not be `mana.ConsensusMana`?

Reply:
Good catch!